### PR TITLE
404 nf challenge

### PR DESCRIPTION
**Pull request by susird**
New pr for 404 not found challenge :sparkle: 
image
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    href="https://fonts.googleapis.com/css2?family=Inconsolata:wght@500&family=Merriweather&family=Montserrat&display=swap"
+    rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Space+Mono:ital,wght@0,400;0,700;1,400;1,700&display=swap"
+    rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <div class="main">
```
Comment (josepplloo) [inline index.html]: Is more useful to have several semantic elements that define the different parts of a web page instead classes, change 
```
<div class="main">
```
for
```
<header >
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <div class="not-found">
```
Comment (josepplloo) [inline index.html]: this `div` is not necessary,  instead use flex/grid for your layout

File: index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      <h1 class="letter-not-found">404 not found</h1>
```
Comment (josepplloo) [inline index.html]: `letter-not-found` is not a useful name for this, try `title` or a different Block/Element name

File: index.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    </div>
+  </div>
+  <div class="body">
```
Comment (josepplloo) [inline index.html]: could be `<section>`?

File: index.html [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    <div class="scarecrow"></div>
```
Comment (josepplloo) [inline index.html]: you miss accesibility `<div role="img" aria-label="Description of the overall image">`

File: index.html [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    <div class="bad-news-title">
+      <h1 class="bad-news"> I have bad news for you</h1>
```
Comment (josepplloo) [inline index.html]: delete white space and you don't have to wrapped this element, also consider rewriting "bad-news-title" class name for `bad-news-container` idem:
"phar-info"
"information"
"button"
"credits"
"credits-creator"
classes

File: index.html [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+    </div>
+    <div class="phar-info">
+      <p class="information">The page you are looking for might be removed or is temporarily unavailable</p>
+    </div>
+    <a href="#" class="button">Back to homepage</a>
+  </div>
+  <div class=" footer">
```
Comment (josepplloo) [inline index.html]: delete white space and use `<footer>` instead of  ` <div class=" footer">`

**Review by josepplloo — CHANGES_REQUESTED** [review]
Very good job 👏 
the changes are about, missing html5 tags, unused classes, and missing/excess styles,
the styles will change, for that reason, I did no put any comments by now...
- try not to use margin properties, first try to solve the issues whit padding!
- if you have 2 o more margins/padding you will use margin property
eg
 ```
    margin-left: 735px;
    margin-top: 265px;
 ```
 === `margin: 265px 0 735px`
  use flex/grid
   - if you have `padding: 0px 0px 24px 0px;` use `padding-bottom: 24px ;` 
   - Pick only one between `px` and `em` (px)
   - don't use decimal `220.35px` -> 220px

File: index.html [hunk 8]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>404 not found</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
```
Comment (anyruizd) [inline index.html]: Do you need both of them?

File: index.html [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+  <link
+    href="https://fonts.googleapis.com/css2?family=Inconsolata:wght@500&family=Merriweather&family=Montserrat&display=swap"
+    rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Space+Mono:ital,wght@0,400;0,700;1,400;1,700&display=swap"
```
Comment (anyruizd) [inline index.html:13]: Are you using the italic variation of the font somewhere? If not, you can delete it 
```suggestion
  <link href="https://fonts.googleapis.com/css2?family=Space+Mono:wght@0,400;0,700;1,400;1,700&display=swap"
```

File: style.css [hunk 10]
```diff
@@ -0,0 +1,144 @@
...
+    background: url("./scarecrow.png") center/cover no-repeat;
+    height: 220.35px;
+    margin-left: 44px;
+    margin-top: 109px;
+    position: absolute;
+    width: 286.83px;
+  }
+
+  .bad-news-title {
+    font-family: Space Mono;
```
Comment (anyruizd) [inline style.css]: You're declaring this `font-family` in many places that are not needed, you should import it just in the outer container of the elements that are using it. You should be using only three font-family definitions, as in the picture. The elements inside the container should inherit the font of their parents.
image

File: style.css [hunk 11]
```diff
@@ -0,0 +1,144 @@
...
+    padding: 0px 0px 24px 0px;
+    position: absolute;
+    width: 287px;
+  }
+
+  @media (min-width: 960px) {
+  .not-found {
+    margin-left: 77px;
+    margin-top: 40px;
+  }
```
Comment (anyruizd) [inline style.css:98]: This media query shouldn't be here, in CSS all media queries go at the bottom of the stylesheet.

File: style.css [hunk 12]
```diff
@@ -0,0 +1,144 @@
...
+    background: url("./scarecrow.png") center/cover no-repeat;
+    height: 447.43px;
+    margin-left: 77px;
+    margin-top: 265px;
+    width: 539.22px;
+  }
+
+  .bad-news {
+    font-size: 64px;
+  }
```
Comment (anyruizd) [inline style.css]: Why this selector only has one property? and why not put that property in the `.bad-news-title` selector? Also, is the title the `h1` element or its container? Why name it the other way around?

File: style.css [hunk 13]
```diff
@@ -0,0 +1,144 @@
...
+
+  .button {
+    margin-left: 740px;
+    margin-top: 663px;
+  }
+
+  .credit-creator { 
+    margin-left: 577px;
+    margin-top: 1039px;
+  }
```
Comment (anyruizd) [inline style.css]: Please ident everything inside the media query

File: style.css [hunk 14]
```diff
@@ -0,0 +1,144 @@
...
+
+  @media (min-width: 960px) {
+  .not-found {
+    margin-left: 77px;
+    margin-top: 40px;
+  }
+
+  .letter-not-found { 
+    font-size: 24px;
+  }
```
Comment (anyruizd) [inline style.css]: same question as in [bad-news ](https://github.com/susird/404-not-found/pull/2/files#r611098885)here

File: style.css [hunk 15]
```diff
@@ -0,0 +1,144 @@
...
+
+  .bad-news-title {
+    font-family: Space Mono;
+    height: 190px;
+    margin-left: 735px;
+    margin-top: 265px;
+    width: 586px;
+  }
+
+  .phar-info {
```
Comment (anyruizd) [inline style.css]: This class name is not meaningful 🤔

File: index.html [hunk 16]
```diff
@@ -0,0 +1,32 @@
...
+      <div class="scarecrow-img"></div>
+    </section>
+   <article class="child-2-container">
+      <h1 class="bad-news-title">I have bad news for you</h1>
+      <p class="info-to-user">The page you are looking for might be removed or is temporarily unavailable</p>
+      <a href="#" class="button">Back to homepage</a>
+    </article>
+  </main>
+</body>
+</html>
```
Comment (josepplloo) [inline index.html]: 👀 insert blank line

File: index.html [hunk 17]
```diff
@@ -0,0 +1,32 @@
...
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <main class="container">
+    <section class="child-1-container">
+      <h1 class="title-not-found">404 not found</h1>
+      <div class="scarecrow-img"></div>
+    </section>
+   <article class="child-2-container">
```
Comment (josepplloo) [inline index.html]: ident this

File: style.css [hunk 18]
```diff
@@ -0,0 +1,104 @@
...
+
+.child-1-container {
+  padding: 20px 0px 50px 24px;
+  font-family: 'Inconsolata';
+}
+
+.title-not-found{
+  font-size: 18px;
+  font-weight: 550;
+  letter-spacing: -0.090em;
```
Comment (josepplloo) [inline style.css]: use px

**Review by josepplloo — CHANGES_REQUESTED** [review]
nice! now looks better but you miss footer

File: style.css [hunk 19]
```diff
@@ -0,0 +1,104 @@
...
+  padding-bottom: 64px;
+}
+
+.scarecrow-img {
+  background: url("./scarecrow.png") center/cover no-repeat;
+  height: 238px;
+  width: 288px;
+}
+
+.child-2-container {
```
Comment (josepplloo) [inline style.css]: you miss display: flex prop

**Review by anyruizd — APPROVED** [review]
I think you don't need this file anymore `scarecrow.png`. Other than that, it looks good to me

**Review by josepplloo — APPROVED** [review]
lgtm 💅

**Pull request merged**